JValldejuli/M2
# Implement gradle

To test:
1. Install gradle
2. Run the gradle commands listed in the pdf and see if they work

EDIT: It seems to all work
